addons/account/models/account_report.py

```diff
@@ -301,6 +302,9 @@ class AccountReportLine(models.Model):
     _order = 'sequence, id'
 
     name = fields.Char(string="Name", translate=True, required=True)
+    json_name = fields.Json(string="Technical Field for the ",
+                            compute='_compute_json_name', #dummy compute
```

Comment:
Either we implement the real compute, or just remove the function. It is useless as is